doc/examples/applications/plot_solidification_tracking.py

```diff
@@ -0,0 +1,226 @@
+    labels={'animation_frame': 'time point'}
+)
+plotly.io.show(fig)
+
+#####################################################################
+# Select largest region
+# =====================
+# In our binary images, the S-L interface appears as the largest region of
+# connected pixels. For this step of the workflow, we will operate on each
+# 2D image separately, as opposed to the entire 3D dataset, because we are
+# only interested in a single moment in time for each region.
+#
+# We select the largest region in each image
+# by computing region properties, including the ``area`` property, and
+# sorting by ``area`` values. Function
+# :func:`skimage.measure.regionprops_table()` returns a table of region
+# properties which can be read into a Pandas ``DataFrame``.
+
+props_0 = measure.regionprops_table(
+        measure.label(binarized[0, :, :]), properties=('label', 'area', 'bbox'))
+props_0_df = pd.DataFrame(props_0)
+props_0_df = props_0_df.sort_values('area', ascending=False)
+# Show top five rows
+props_0_df.head()
+
+#####################################################################
+# We can visualize the largest region in the 0th image with its
+# bounding box (bbox) by first labeling the binary image and selecting
+# the labels that correspond to the largest region.
+
+labeled_0 = measure.label(binarized[0, :, :])
+largest_region_0 = labeled_0 == props_0_df.iloc[0]['label']
+minr, minc, maxr, maxc = [props_0_df.iloc[0][f'bbox-{i}'] for i in range(4)]
+fig = px.imshow(largest_region_0, binary_string=True)
+fig.add_shape(
+        type="rect", x0=minc, y0=minr, x1=maxc, y1=maxr, line=dict(color="Red"))
+plotly.io.show(fig)
+
+#####################################################################
+# Next we find the largest region in each image for all images in the
+# dataset into a 3D NumPy array ``largest_region``. We'll also store
+# the bbox information for each image, which will be used to track the
+# position of the S-L interface.
+
+largest_region = np.empty_like(binarized)
+bboxes = []
+
+for i in range(binarized.shape[0]):
+    labeled = measure.label(binarized[i, :, :])
+    props = measure.regionprops_table(
+            labeled, properties=('label', 'area', 'bbox'))
+    props_df = pd.DataFrame(props)
+    props_df = props_df.sort_values('area', ascending=False)
+    largest_region[i, :, :] = (labeled == props_df.iloc[0]['label'])
+    bboxes.append([props_df.iloc[0][f'bbox-{i}'] for i in range(4)])
+fig = px.imshow(
+    largest_region,
+    animation_frame=0,
+    binary_string=True,
+    labels={'animation_frame': 'time point'}
+)
+plotly.io.show(fig)
+
+#####################################################################
+# Plot interface location over time
+# =================================
+# The final step in this analysis is to plot the location of the S-L
+# interfaces over time. This can be achieved by plotting ``maxr``
+# over time since this value shows the y location of the bottom of
```

Comment:
```suggestion
# interfaces over time. This can be achieved by plotting ``maxr``
# (third element in bbox) 
# over time since this value shows the `y` location of the bottom of
```